### PR TITLE
Integrate FINOS Governance into Project Blueprint CONTRIBUTING Files

### LINKED ISSUE #17: Project blueprint should have the new governance built in
- [x] Check if CONTRIBUTING.md works on root folder
- [x] Merge CONTRIBUTING.template.md into CONTRIBUTING.md
- [x] Update title to Software Project Blueprint
- [x] "This blueprint implements https://github.com/finos/community/tree/master/governance#open-source-software-projects" into the README.md of this repo
- [x] Arrange placeholders
- [x] The new governance allows for a governance for software projects (https://github.com/finos/community/blob/master/governance/Software-Projects/CONTRIBUTING.template.md) and one for standard projects (https://github.com/finos/community/blob/master/governance/Standards-Projects/CONTRIBUTING.standards.md).

For now, I suggest you have both of them in the blueprint and then remember to select the correct one when you instantiate a new project. 

Once you are done can you also send a pull request to Morphir and Glue to adopt it (apologizing it wasn't done in the first place)?

### PR DESCRIPTION
**Pull request by mcleo-d**
## Description

This pull request Closes #17 with the following commits added to the contribution ...

- Add governance reference to project `README.md` per issue #17 4729dfa
- Move `CONTRIBUTING.md` from `.github` folder to project `root` per issue #17 4a2362c and 7a9bb2c
- Integration of https://github.com/finos/community/blob/master/governance/Software-Projects/CONTRIBUTING.template.md into main project contributing file per issue #17 9fee2d5
- Add https://raw.githubusercontent.com/finos/community/master/governance/Software-Projects/CONTRIBUTING.template.md to project `root` per #17 3130eff
- Add https://github.com/finos/community/blob/master/governance/Standards-Projects/CONTRIBUTING.standards.md to project `root` per #17 bc968b7 
- Update [PROJECT] to {project name} a8e0ff0 and 2112980

### CODE REVIEW

File: CONTRIBUTING.template.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# [PROJECT] Contribution and Governance Policies
```
Comment (maoo) [inline CONTRIBUTING.template.md:1]: Could you please use `{project name}` instead of `[PROJECT]` ?

File: CONTRIBUTING.standards.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
+# [Project Name] Charter
```
Comment (maoo) [inline CONTRIBUTING.standards.md:1]: Could you please use `{project name}` instead of `[Project Name]` ?

File: CONTRIBUTING.template.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+This document describes the contribution process and governance policies of the FINOS [PROJECT] project. The project is also governed by the [Linux Foundation Antitrust Policy](https://www.linuxfoundation.org/antitrust-policy/), and the FINOS [IP Policy](IP-Policy.pdf), [Code of Conduct](Code-of-Conduct.md), [Collaborative Principles](Collaborative-Principles.md), and [Meeting Procedures](Meeting-Procedures.md).
```
Comment (maoo) [inline CONTRIBUTING.template.md:3]: Could you please use `{project name}` instead of `[PROJECT]` ?

**Review by maoo — CHANGES_REQUESTED** [review]
Requested few changes to placeholders, other than that, LGTM

File: CONTRIBUTING.template.md (hunk 1 above)
Comment (mcleo-d) [reply CONTRIBUTING.template.md]: Done 👍

File: CONTRIBUTING.standards.md (hunk 2 above)
Comment (mcleo-d) [reply CONTRIBUTING.standards.md]: Done 👍

File: CONTRIBUTING.template.md (hunk 3 above)
Comment (mcleo-d) [reply CONTRIBUTING.template.md]: Done 👍

**Review by maoo — APPROVED** [review]
TY @mcleo-d !

**Pull request merged**